From 11884a67cf26ed46e967dca9dee5e6fa32eb92b6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B6rn=20Esser?= <besser82@fedoraproject.org>
Date: Wed, 19 Apr 2017 19:08:35 +0200
Subject: [PATCH] Replace gtk_widget_get_root_window()

---
 src/ygtkmenubutton.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ygtkmenubutton.c b/src/ygtkmenubutton.c
index 22c96a6..5ed81a5 100644
--- a/src/ygtkmenubutton.c
+++ b/src/ygtkmenubutton.c
@@ -76,7 +76,7 @@ static void ygtk_popup_window_frame_position (GtkWidget *widget, gint *x,  gint
 	gtk_widget_get_preferred_size(widget, &req, NULL);
 
 	GdkScreen *screen = gtk_widget_get_screen (widget);
-	gint monitor_num = gdk_screen_get_monitor_at_window (screen, gtk_widget_get_root_window (widget));
+	gint monitor_num = gdk_screen_get_monitor_at_window (screen, gdk_screen_get_root_window (gtk_widget_get_screen (widget)));
 	GdkRectangle monitor;
 	gdk_screen_get_monitor_geometry (screen, monitor_num, &monitor);
 
