From cc3ac74b1186b33bd9e6dcd532e189b5b4cb29bb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B6rn=20Esser?= <besser82@fedoraproject.org>
Date: Wed, 19 Apr 2017 11:04:39 +0200
Subject: [PATCH] Fix Werror=misleading-indentation

---
 src/YGDialog.cc | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/YGDialog.cc b/src/YGDialog.cc
index 0e26534..cccfaf2 100644
--- a/src/YGDialog.cc
+++ b/src/YGDialog.cc
@@ -127,17 +127,17 @@ class YGWindow
 		    	if (YGUI::ui()->isSwsingle())
 		    		height += YGUtils::getCharsHeight (m_widget, 10);
 
-				width = MIN (width, YUI::app()->displayWidth());
-				height = MIN (height, YUI::app()->displayHeight());
+			width = MIN (width, YUI::app()->displayWidth());
+			height = MIN (height, YUI::app()->displayHeight());
 
 		        gtk_window_set_default_size (window, width, height);
 		        gtk_window_resize(window, width, height);
 
-				if (YGUI::ui()->setFullscreen())
-					gtk_window_fullscreen (window);
-				else if (YUI::app()->displayWidth() <= 800 || YUI::app()->displayHeight() <= 600)
-					// maximize window for small displays
-					gtk_window_maximize (window);
+			if (YGUI::ui()->setFullscreen())
+				gtk_window_fullscreen (window);
+			else if (YUI::app()->displayWidth() <= 800 || YUI::app()->displayHeight() <= 600)
+				// maximize window for small displays
+				gtk_window_maximize (window);
 		    }
 
 		    gtk_window_set_role (window, "yast2");
