From 355cbac51273f87b68e19430f591cee2c16766ed Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B6rn=20Esser?= <besser82@fedoraproject.org>
Date: Wed, 19 Apr 2017 21:33:25 +0200
Subject: [PATCH] Remove gtk_tree_view_set_rules_hint()

---
 src/YGTreeView.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/YGTreeView.cc b/src/YGTreeView.cc
index bb62a53..be4c8ff 100644
--- a/src/YGTreeView.cc
+++ b/src/YGTreeView.cc
@@ -354,7 +354,7 @@ class YGTable : public YTable, public YGTreeView
 	  YGTreeView (this, parent, std::string(), false)
 	{
 		gtk_tree_view_set_headers_visible (getView(), TRUE);
-		gtk_tree_view_set_rules_hint (getView(), columns() > 1);
+		// gtk_tree_view_set_rules_hint (getView(), columns() > 1);
 		ygtk_tree_view_set_empty_text (YGTK_TREE_VIEW (getView()), _("No entries."));
 		if (multiSelection)
 			gtk_tree_selection_set_mode (getSelection(), GTK_SELECTION_MULTIPLE);
