From 21ecc082bdf5628b91bb844e6993de4789e5d15d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B6rn=20Esser?= <besser82@fedoraproject.org>
Date: Wed, 19 Apr 2017 15:38:50 +0200
Subject: [PATCH] Remove obsolete gtk_window_set_has_resize_grip()

---
 src/YGDialog.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/YGDialog.cc b/src/YGDialog.cc
index 5d7ce55..8649dba 100644
--- a/src/YGDialog.cc
+++ b/src/YGDialog.cc
@@ -54,7 +54,7 @@ class YGWindow
 		m_widget = ygtk_window_new();
 		// gtk_container_set_resize_mode (GTK_CONTAINER (m_widget), GTK_RESIZE_PARENT);
 		g_object_ref_sink (G_OBJECT (m_widget));
-		gtk_window_set_has_resize_grip (GTK_WINDOW (m_widget), TRUE);
+		// gtk_window_set_has_resize_grip (GTK_WINDOW (m_widget), TRUE);
 
 		m_refcount = 0;
 		m_child = NULL;
