From 5077aae7d96d607d96600a38484384ca8e5a0dee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B6rn=20Esser?= <besser82@fedoraproject.org>
Date: Wed, 19 Apr 2017 13:32:08 +0200
Subject: [PATCH] Replace gtk_style_context_get_background_color()

---
 src/ygtkbargraph.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/ygtkbargraph.c b/src/ygtkbargraph.c
index c96fc02..0b6d08c 100644
--- a/src/ygtkbargraph.c
+++ b/src/ygtkbargraph.c
@@ -190,7 +190,8 @@ static gboolean ygtk_colored_label_on_draw (GtkWidget *widget, cairo_t *cr)
 	ctx = gtk_widget_get_style_context(widget);
 
 	GdkRGBA color;
-	gtk_style_context_get_background_color(ctx, GTK_STATE_NORMAL, &color);
+	gtk_style_context_get (ctx, GTK_STATE_NORMAL,
+			       "background-color", &color, NULL);
 
     cairo_save(cr);
     int width = gtk_widget_get_allocated_width (widget);
